ui/components/component-library/banner-tip/banner-tip.stories.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from 'react';
+import {
+  DISPLAY,
+  FLEX_DIRECTION,
+  Size,
+} from '../../../helpers/constants/design-system';
+import Box from '../../ui/box/box';
+import { ButtonLink, ButtonPrimary, ICON_NAMES } from '..';
+import README from './README.mdx';
+import { BannerTip, BANNER_TIP_LOGOS } from '.';
+
+const marginSizeControlOptions = [
+  undefined,
+  0,
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9,
+  10,
+  11,
+  12,
+  'auto',
+];
+
+export default {
+  title: 'Components/ComponentLibrary/BannerTip',
+  component: BannerTip,
+  parameters: {
+    docs: {
+      page: README,
+    },
+    backgrounds: { default: 'alternative' },
+  },
+  argTypes: {
+    logoType: {
+      options: Object.values(BANNER_TIP_LOGOS),
+      control: 'select',
+    },
+    className: {
+      control: 'text',
+    },
+    title: {
+      control: 'text',
+    },
+    description: {
+      control: 'text',
+    },
+    children: {
+      control: 'text',
+    },
+    action: {
+      control: 'func',
+    },
+    actionButtonLabel: {
+      control: 'text',
+    },
+    actionButtonOnClick: {
+      control: 'func',
+    },
+    actionButtonProps: {
+      control: 'object',
+    },
+    onClose: {
+      action: 'onClose',
+    },
+    marginTop: {
```

Comment:
Do we need to add any of these under `table: 'banner base props'`?
Kinda like this - reminder need to update to remove "base"
<img width="1440" alt="Screenshot 2023-02-22 at 5 02 44 PM" src="https://user-images.githubusercontent.com/8112138/220791697-49ff1e41-91fd-43c0-bd37-c325710f0aa8.png">